relational_operators/DropTableOperator.cpp

```diff
@@ -32,9 +34,21 @@ namespace quickstep {
 bool DropTableOperator::getAllWorkOrders(WorkOrdersContainer *container) {
   if (blocking_dependencies_met_ && !work_generated_) {
     work_generated_ = true;
+
+    std::vector<block_id> relation_blocks(relation_.getBlocksSnapshot());
+
+    // DropTableWorkOrder only drops blocks, if any.
     container->addNormalWorkOrder(
-        new DropTableWorkOrder(relation_.getID(), only_drop_blocks_),
+        new DropTableWorkOrder(std::move(relation_blocks)),
```

Comment:
I don't think so. Passing a const reference would result in a deep copy. In other words, there will be two copies of the contents, instead of one in the destination.